bundles/org.openhab.binding.orbitbhyve/src/main/feature/feature.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.orbitbhyve-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+	<repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
+
+	<feature name="openhab-binding-orbitbhyve" description="OrbitBhyve Binding" version="${project.version}">
```

Comment:
This name will be displayed, when installing the binding via the UI.
```suggestion
	<feature name="openhab-binding-orbitbhyve" description="Orbit B-Hyve Binding" version="${project.version}">
```

Reply:
Fixed